frontend/components/shared/buttons/BackToPageButton.tsx

```diff
@@ -0,0 +1,13 @@
+import Link from 'next/link';
+type BackToPageProps = {
+  text: string;
+  href: string | URL;
+};
+
+const BackToPageButton = ({ text, href }: BackToPageProps) => (
```

Comment:
To be honest, this button looks useless, you should be able to provide styles from here to Button component and use Button component instead of this one, since Button has the same props as here

Reply:
You are right, but I wanted to have this button ready to be used without write again styles. And this is not really button like another so I think I will keep it that way